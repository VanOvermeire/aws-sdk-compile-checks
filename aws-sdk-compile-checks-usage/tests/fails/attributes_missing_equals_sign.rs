use aws_sdk_compile_checks_macro::required_props;

#[required_props(sdk something)]
async fn do_call() {
    // irrelevant
}

fn main() {}
