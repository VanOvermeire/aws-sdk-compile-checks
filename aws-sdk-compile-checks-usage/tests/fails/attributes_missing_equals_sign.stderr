error: expected `sdk` to be followed by a `=` and one or more SDKs, e.g. `sdk = sqs`
 --> tests/fails/attributes_missing_equals_sign.rs:3:18
  |
3 | #[required_props(sdk something)]
  |                  ^^^
